sdcm/nemesis.py

```diff
@@ -6234,3 +6264,10 @@ class BootstrapStreamingErrorNemesis(Nemesis):
 
     def disrupt(self):
         self.disrupt_bootstrap_streaming_error()
+
+
+class DisableBinaryGossip(Nemesis):
```

Comment:
Please change the name to reflect the major compaction

Reply:
Done